src/components/UploadingModal/UploadingModal.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import UploadingSpinner from '../UploadingSpinner/UploadingSpinner'
+import './UploadingModal.scss'
+
+const UploadingModal = props => {
```

Comment:
we should destructure props here to be consistent with other components - something to hit when we go through and refactor.